pkg/ruler/manager.go

```diff
@@ -20,6 +20,24 @@ import (
 	"github.com/cortexproject/cortex/pkg/util"
 )
 
+var (
+	configUpdatesTotal = promauto.NewCounterVec(prometheus.CounterOpts{
```

Comment:
I would move back `configUpdatesTotal` and `configUpdateFailuresTotal` to the ruler source file because they belong to the ruler, not the manager.